notifications_utils/template.py

```diff
@@ -572,7 +573,7 @@ def _contact_block(self):
 
     @property
     def _date(self):
-        return self.date.strftime('%-d %B %Y')
+        return self.date.strftime('{} %B %Y'.format(no_pad_day()))
```

Comment:
```suggestion
        return self.date.strftime(f'no_pad_day() %B %Y')
```
f-strings yay

Reply:
You're really a big fan of the f-strings. 😉